bom/pom.xml

```diff
@@ -232,7 +232,7 @@ THE SOFTWARE.
       <dependency>
         <groupId>antlr</groupId>
         <artifactId>antlr</artifactId>
-        <version>2.7.6</version>
+        <version>2.7.7</version>
```

Comment:
Lol this get's us from 2006, to 2007.

they changed the group ID after that https://mvnrepository.com/artifact/org.antlr/antlr4

Reply:
I had a brief look at changing but there's breaking changes and I know nothing about antlr :p